tests/test_wave_03.py

```diff
@@ -128,13 +128,13 @@ def test_mark_complete_missing_task(client):
     # Assert
     assert response.status_code == 404
 
-    raise Exception("Complete test with assertion about response body")
+    # raise Exception("Complete test with assertion about response body")
     # *****************************************************************
     # **Complete test with assertion about response body***************
     # *****************************************************************
+    assert response_body == {"message" : "Task 1 is not found"}
```

Comment:
Same comment as the ones I left in test_wave_01.py -- double check your assertion message here. The test fails but only because response_body doesn't match the message by one word.

In industry, when you submit a PR for review, you'll need to ensure that all the tests pass. Usually teams don't review code that has failing tests